app/services/notification/Notifications.scala

```diff
@@ -0,0 +1,63 @@
+package services.notification
+
+import javax.inject.{Inject, Singleton}
+
+import config.{AMIableConfig, AmiableConfigProvider}
+import org.quartz.CronScheduleBuilder.cronSchedule
+import org.quartz.JobBuilder.newJob
+import org.quartz.{JobKey, TriggerKey}
+import org.quartz.TriggerBuilder.newTrigger
+import org.quartz.impl.StdSchedulerFactory
+import play.api.inject.ApplicationLifecycle
+import play.api.{Configuration, Environment, Logger, Mode}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class Notifications @Inject()(
+                               amiableConfigProvider: AmiableConfigProvider,
+                               environment: Environment,
+                               lifecycle: ApplicationLifecycle,
+                               mailClient: AWSMailClient)
+                             (implicit exec: ExecutionContext,
+                              configuration: Configuration) {
+  implicit val conf: AMIableConfig = amiableConfigProvider.conf
+  /**
+    * a Quartz cron expression,
+    * e.g. "0 0 3 * * ? *" to run at 3am every day
+    * See http://www.quartz-scheduler.org/documentation/quartz-2.2.x/tutorials/crontrigger.html
+    */
+  private val ownerSchdlCron = configuration.getString("amiable.owner.notification.cron").get
+  private val scheduler = StdSchedulerFactory.getDefaultScheduler
+
+  // send notifications only in PROD
+  if (environment.mode == Mode.Prod) {
+    Logger.info("Starting the scheduler")
```

Comment:
Might be worth saying what this scheduler is for in the log message.

Really minor stuff again!